app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/EnabledMessageComposer.kt

```diff
@@ -301,11 +301,11 @@ fun EnabledMessageComposer(
                         if (additionalOptionStateHolder.selectedOption == AdditionalOptionSelectItem.DrawingMode) {
                             DrawingCanvasBottomSheet(
                                 onDismissSketch = {
-                                    inputStateHolder.collapseComposer(additionalOptionStateHolder.additionalOptionsSubMenuState)
+                                    showAdditionalOptionsMenu()
```

Comment:
Just delegate all the orchestration to the function, instead of manually collapsing the keyboard